include/deal.II/lac/read_write_vector.templates.h

```diff
@@ -164,11 +165,12 @@ namespace LinearAlgebra
 #ifdef DEAL_II_WITH_TRILINOS
   template <typename Number>
   ReadWriteVector<Number> &
-  ReadWriteVector<Number>::operator = (const TrilinosWrappers::MPI::Vector &trilinos_vec)
+  ReadWriteVector<Number>::import(const Epetra_MultiVector &multivector,
```

Comment:
Question: What happens if the multivector indeed is a multi-vector, i.e., it holds more than one column? Should we insert an assertion?